easybuild/easyconfigs/a/ant/ant-1.10.5-Java-1.8.eb

```diff
@@ -1,24 +1,29 @@
-easyblock = 'PackedBinary'
-
 name = 'ant'
 version = '1.10.5'
 versionsuffix = '-Java-%(javaver)s'
 
-homepage = 'http://ant.apache.org/'
+homepage = 'https://ant.apache.org/'
 description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in
  build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of
  Java applications."""
 
 toolchain = SYSTEM
 
-source_urls = ['http://apache.cu.be//ant/binaries']
-sources = ['apache-%(name)s-%(version)s-bin.tar.gz']
-checksums = ['cd52c6f77a0b1de7174addc94e5a512d86978ac9173a30cd11f4ab26f6d8deba']
+source_urls = ['https://archive.apache.org/dist/%(name)s/source/']
+sources = ['apache-%(name)s-%(version)s-src.tar.gz']
+patches = ['ant-fix-doclint-error.patch']
+checksums = [
+    '5937cf11d74d75d6e8927402950b012e037e362f9f728262ce432ad289b9f6ca',  # source
+    '28093a4643bdecf690b528a572b1ec7243c000f427c785158eb9d8c957fc222b',  # ant-fix-doclint-error.patch
+]
+
+# specify dependency on Java/1.8 "wrapper", rather than a specific Java version
+dependencies = [('Java', '1.8', '', True)]
 
-dependencies = [('Java', '1.8')]
+builddependencies = [('JUnit', '4.12', versionsuffix)]
 
 sanity_check_paths = {
-    'files': ['bin/ant', 'lib/ant.jar'],
+    'files': ['bin/ant', 'lib/ant.jar', 'lib/ant.jar'],
```

Comment:
@Flamefire Why check the same file twice?

Reply:
C&P from old one :/